mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Realtime.MixProject do
   def project do
     [
       app: :realtime,
-      version: "2.29.8",
+      version: "2.29.9",
```

Comment:
we released today, move it to `2.29.10` please 🙏

Reply:
just bumped @filipecabaco :)